client/js/analysis/gdpr.js

```diff
@@ -67,14 +79,117 @@ export async function getStreamingData(data) {
             Promise
                 .all(contentPromises)
                 .then((contentWithIndices) => {
-                  const content = contentWithIndices
-                      .map((entry) => entry.data)
-                      .reduce((dataA, dataB) => dataA.concat(dataB), []);
-                  resolve(content);
+                  resolve(
+                      contentWithIndices
+                          .map((entry) => entry.data)
+                          .reduce((dataA, dataB) => dataA.concat(dataB), [])
+                          .map((entry) => {
+                            // dates are strings in JSON, convert to JS date
+                            const endTime = new Date(entry.endTime);
+
+                            // move according to UTC offset
+                            const endTimeMs = +endTime;
+                            const endTimeOffset =
```

Comment:
nit: name this endTimeOffsetMs to let folks know how you're intending to change the units of getTimezoneOffset. Unfortunately the fact that getTimezoneOffset returns a number in minutes isn't clear from the API name, but variable names like this help clarify this as well as your intentions.